docs/config/node.rst

```diff
@@ -501,6 +501,27 @@ long. The following settings can be used to adjust these timeouts:
 Authentication
 ==============
 
+.. _external_auth:
+
+External Authentication
```

Comment:
Isn't this very specific to JWT?
```suggestion
JWT Based Authentication
```

Reply:
Applied and since those settings are part of HBA setting, I think I shouldn't even introduce new section [here](https://cratedb.com/docs/crate/reference/en/latest/config/node.html#authentication).

I will move them under HBA (to be grouped with other auth.host_based.* settings)